modules/howtos/pages/managing-connections.adoc

```diff
@@ -147,11 +148,30 @@ NOTE: Any TLS certificates must be set up at the point where the connections are
 [#ssl]
 == Secure Connections
 
-Couchbase Server Enterprise Edition supports full encryption of client-side traffic using Transport Layer Security (TLS).
+Couchbase Server Enterprise Edition and Couchbase Capella support full encryption of client-side traffic using Transport Layer Security (TLS).
 That includes key-value type operations, queries, and configuration communication.
-Make sure you have the Enterprise Edition of Couchbase Server before proceeding with configuring encryption on the client side.
+Make sure you have the Enterprise Edition of Couchbase Server, or a Couchbase Capella account, before proceeding with configuring encryption on the client side.
+
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+The Java SDK includes Capella’s standard root certificate by default, which means you don’t need any additional configuration to enable TLS.
+Simply use `couchbases://` in your connection string, or `SecurityConfig.enableTls(true)` in your `SecurityConfig` settings.
+--
+
+Couchbase Server::
++
+--
+As of SDK 3.4, if your cluster's root certificate is signed by a well known Certificate Authority (e.g., GoDaddy, Verisign, etc) and is included in the JVM's trust store, you no longer need to configure the certificate in the `securityConfig` settings.
```

Comment:
> and is included in the JVM's trust store,

In the case we're talking about here, the cluster's root certificate doesn't need to be in the JVM's trust store. The cluster's root certificate just needs to be *issued* (signed) by a CA whose certificate is in the JVM's trust store.

Another detail that might be useful: when we talking about the JVM's trust store, we're specifically referring to the `cacerts` file.